src/test/java/com/exasol/adapter/dynamodb/OpenLibrary.java

```diff
@@ -0,0 +1,39 @@
+package com.exasol.adapter.dynamodb;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.FileChannel;
+import java.nio.channels.ReadableByteChannel;
+
+/**
+ * This class imports the open library dataset into a DynamoDB table.
+ */
+public class OpenLibrary {
+    private static final String DATASET_URL = "https://s3-eu-west-1.amazonaws.com/csparkdata/ol_cdump.json";
+    private static final String TMP_FILE_NAME = "/tmp/dataset.json";
+    private static final String TABLE_NAME = "open_library_test";
+    private final DynamodbTestInterface dynamodbTestInterface;
+
+    public OpenLibrary(final DynamodbTestInterface dynamodbTestInterface) throws IOException {
+        this.dynamodbTestInterface = dynamodbTestInterface;
+        if (dynamodbTestInterface.isTableEmpty(TABLE_NAME)) {
```

Comment:
Create another public method, setupDataset or createDataset and move these two methods there.

Reply:
I will did already prepare a comple restructuring of this whole class in a second branch. Therefore i'll leave it as it is here to avoid merge conflicts.

Reply:
Okay, sure.